src/main/scala/io/scala/models/Session.scala

```diff
@@ -1,62 +1,57 @@
 package io.scala.models
 
 import com.raquo.laminar.api.L.*
-import com.raquo.laminar.nodes.ReactiveHtmlElement
+import com.raquo.laminar.nodes.{ReactiveHtmlElement, ReactiveSvgElement}
 import java.time.DayOfWeek
 import java.time.LocalDateTime
 import java.time.LocalTime
 import org.scalajs.dom
-import org.scalajs.dom.HTMLDivElement
+import org.scalajs.dom.{HTMLDivElement, HTMLParagraphElement, SVGSVGElement}
 
 import io.scala.data.current
 import io.scala.data.parsers.Parsers
 import io.scala.extensions.*
-import io.scala.modules.TalkCard
+import io.scala.modules.SessionCard
 import io.scala.modules.elements.Paragraphs
 import io.scala.svgs.Icons
 
-sealed trait TalkInfo[A <: TalkInfo[A]]:
-  def ordinal: Int
-
-object TalkInfo:
-  given [A <: TalkInfo[A]]: Ordering[A] = Ordering[Int].on(_.ordinal)
-
-type CompleteAct = Act & { val time: LocalTime; val day: DayOfWeek }
 sealed trait Act:
-  def day: DayOfWeek | Null
-  def time: LocalTime | Null
+  def day: DayOfWeek
+  def time: LocalTime
```

Comment:
Hum, it still compiles with this removed ?

Reply:
It appears so, you might have found a simpler way to do it

Reply:
Every concrete subtype of `Act` had `day` and `time` defined, so they we're already "complete" 😉

Reply:
Not necessarily, currently none of them has a non null value (or at least that's what I expect). But since the schedule is not available it's not an issue atm